dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/XtxtUML.xtext

```diff
@@ -212,10 +212,18 @@ TUMultiplicity :
 
 // Commons
 
+TUModifiers:
```

Comment:
I agree with your points and I've totally forgot about the customizability of syntax error messages. The implementation can stay as it is now, and we should customize the error messages later.